File: alarmclock/src/test/java/com/externalalarmclock/alarmclock/healthchecks/AlarmClockHealthCheckTest.java
package com.externalalarmclock.alarmclock.healthchecks;

import org.junit.Assert;
import org.junit.Test;

import com.codahale.metrics.health.HealthCheck.Result;

public class AlarmClockHealthCheckTest {
	@Test
	public void testCheck() throws Exception {
		Result result = new AlarmClockHealthCheck().check();
		
		Assert.assertTrue(result.isHealthy());
	}
}

File: alarmclock/src/test/kotlin/com/externalalarmclock/alarmclock/healthchecks/AlarmClockHealthCheckTest.kt
package com.externalalarmclock.alarmclock.healthchecks

import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Test

class AlarmClockHealthCheckTest {
    @Test
    fun testCheck() {
        val result = AlarmClockHealthCheck().check()

        Assertions.assertTrue(result.isHealthy)
    }
}
